tests/test_logical_types.py

```diff
@@ -474,3 +475,80 @@ def test_local_timestamp_micros():
     binary = serialize(schema, 3600 * 1000 * 1000)
     data2 = deserialize(schema, binary)
     assert tz_naive == data2
+
+
+class Interface:
+    def __init__(self, array_interface):
+        array_interface["shape"] = tuple(array_interface["shape"])
+        self.__array_interface__ = array_interface
+
+
+def read_ndarray(data, writer_schema, reader_schema):
+    import numpy as np
```

Comment:
This can be removed since the import is at the top of the file.

Reply:
ah, this was copy pasted from some code that I wanted to register the logical type conversion, but not import numpy until it actually needed it